en/django_templates/README.md

```diff
@@ -19,7 +19,7 @@ To print a variable in Django templates, we use double curly brackets with the v
 {{ posts }}
 ```
 
-Try this in your `blog/templates/blog/post_list.html` template. Open it up in the code editor, and replace everything from the second `<div>` to the third `</div>` with `{{ posts }}`. Save the file, and refresh the page to see the results:
+Try this in your `blog/templates/blog/post_list.html` template. Open it up in the code editor, and replace the existing `<article>` blocks with `{{ posts }}`. Save the file, and refresh the page to see the results:
```

Comment:
Let's use the proper terminology:
```suggestion
Try this in your `blog/templates/blog/post_list.html` template. Open it up in the code editor, and replace the existing `<article>` elements with `{{ posts }}`. Save the file, and refresh the page to see the results:
```

("blocks" would be doubly confusing, as Django's templates have a concept named like that.)